toolchain.mk

```diff
@@ -1,6 +1,7 @@
 ################################################################################
 # Toolchains
 ################################################################################
+SHELL				:= /bin/bash
```

Comment:
Please use `=` instead of `:=` because there is no reason for `:=`.

Reply:
thanks @jforissier . take your advice